app/src/org/commcare/activities/AdvancedActionsActivity.java

```diff
@@ -223,9 +226,9 @@ private void startConnectionTest() {
         startActivity(i);
     }
 
-    private void clearUserData() {
+    public static void clearUserData(final Activity activity) {
```

Comment:
I'm curious, does this UX extension has anything to do with the offline restore feature?